packages/grafana-data/src/transformations/transformers/reduce.ts

```diff
@@ -59,13 +62,20 @@ export const reduceTransformer: DataTransformerInfo<ReduceTransformerOptions> =
         }
         for (let i = 0; i < series.fields.length; i++) {
           const field = series.fields[i];
+          if (field.type === FieldType.time) {
+            continue;
+          }
+
           if (matcher(field)) {
             const results = reduceField({
               field,
               reducers,
             });
             // Update the name list
-            values[0].buffer.push(field.name);
+            const seriesName = series.name ?? series.refId ?? seriesIndex;
+            const fieldName =
+              field.name === seriesName || data.length === 1 ? field.name : `${field.name} {${seriesName}}`;
+            values[0].buffer.push(fieldName);
```

Comment:
This is the only part I am not wild about...  I would rather have two fields, one for the series name and one for the field name.  The modified name seems like it will just be awkward

Reply:
Yes I get that, but it also introduces another table column, so that's why I did it :)

Reply:
is another column bad?

Reply:
no worries I'll change it and add another column. Thanks for the feedback 🙌

Reply:
Yeah, I see your point @ryantxu , but this will result in table displaying one additional column that will be required to be filtered out by another transformer. Don't have a strong opinion on this one, but apart from the complex logic above I think this implementation sort of makes sense.

Reply:
After discussing with @dprokop I will merge this as is. If we want to change this later, it's an easy change to do.